org.dupot.nix-disk-manager.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.dupot.nix-disk-manager
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: nix_disk_manager
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=xdg-documents
+  - --filesystem=host-etc
```

Comment:
why `host-etc`?

Reply:
Hi,
This application will write in /etc/nixos/hardware-configuration.nix
this is the nixos configuration file to manage disk drive

Reply:
So this app only works in NixOS?

Reply:
yes, do i need to specify it and where ?

Reply:
Add it to the metainfo description as the first `<p>` tag.

Reply:
ok thanks, done

Reply:
then this is not suitable for flathub. Flatpak is meant to be universal.

Reply:
Yea I agree this is better published as a nix flake. It doesn't make sense for Flathub to host an app when it is tied to a specific distro

Reply:
Ok, you can cancel this request, thanks for your help.